hasura.planx.uk/migrations/1705407249614_squashed/up.sql

```diff
@@ -0,0 +1,12 @@
+
+CREATE TABLE "public"."feedback_type_enum" ("value" text NOT NULL, "comment" text NOT NULL, PRIMARY KEY ("value") );COMMENT ON TABLE "public"."feedback_type_enum" IS E'Store the different types of feedback we gather from user';
+
+INSERT INTO "public"."feedback_type_enum"("value", "comment") VALUES (E'issue', E' A user reporting an issue with a service');
```

Comment:
```suggestion
INSERT INTO "public"."feedback_type_enum"("value", "comment") VALUES (E'issue', E'A user reporting an issue with a service');
```

Reply:
I've made this change on another branch so this comment hasn't been lost 😌